Makefile

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/make -f
+# Makefile for FluidPlug #
```

Comment:
The what? =)

Reply:
lol copy&paste silliness, you didn't see anything :rainbow: